### DISCUSSION TITLE
Cut a release?

### DISCUSSION THREAD

**Discussion by FranklinYu** [zdharma-continuum/zinit#344] [Q&A] [answered]
Last tag (v3.7) was cut in December 2020; since then, there has been 446 commits in the main branch: https://github.com/zdharma-continuum/zinit/compare/v3.7...main

Is it worth cutting a new release? There isn’t anything I’m waiting for so far, but that’s probably because I’m new here.

**Comment by vladdoster** [comment]
Yes, you are 101% correct. It will be a major version bump (i.e., `v4.0.0`). 

I will add the tooling for cutting releases.

I envision it will update the following:

- Documentation site (WIP -- but here is a [sneak peek](https://zinit.readme.io) ;) ) 
- HomeBrew formula
- Docker images
- GH Release
- Changelog
 
Thanks for bringing it up. I'll create an issue to track the progress.

**Comment by vladdoster** [comment, marked as answer]
@FranklinYu Issue #349 for tracking progress 👍🏻.

**Reply by vladdoster** [reply]
@FranklinYu, [v3.8.0](https://github.com/zdharma-continuum/zinit/releases/tag/v3.8.0) was just released!
